packages/runner/bin/runner.js

```diff
@@ -48,20 +52,13 @@ import setupServer from '../src/setupServer.js';
   console.log(chalk.gray('Starting runner with your game...\n'));
   console.log('running with options:', options);
 
-  const cleanupServerFunc = await setupServer({
-    isEmptyBackend: options.emptyBackend,
-    apiPort: portForRunnerBackend,
-  });
+  const cleanupServerFunc = await setupServer({ apiPort: portForRunnerBackend });
 
   let cleanupFrontendsFunc;
   let runnerFrontendProcess;
-  let userFrontendProcess;
 
   if (options.dev) {
-    userFrontendProcess = exec(`cd test_app/frontend && BROWSER=None PORT=${portForUserFrontend} npm start`);
```

Comment:
No need for the userFrontendProcess anymore because we expect the `dev` command in the `test-app` package to run `npm start` for the user frontend (exactly like how we would do in the `tictactoe` example.